### PR TITLE
Feat/tgy

### PR DESCRIPTION
**Pull request by YoNG-Zaii**
Markdown changes

### CODE REVIEW

**Comment by chanbakjsd** [comment]
Any reason why it's two commits instead of one?

**Review by limivann — APPROVED** [review]
👀

**Review by limivann — COMMENTED** [review]
Why empty file???

**Comment by YoNG-Zaii** [comment]
> 👀

Hello, I'm still working on it.

File: pages/Testing.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Testing
 
-Just trying something new
```
Comment (bleow) [inline pages/Testing.md]: maybe change ver to new!

**Review by bleow — CHANGES_REQUESTED** [review]

**Comment by SLAU925** [comment]
> Why empty file???

File: pages/Testing.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Testing
```
Comment (bleow) [inline pages/Testing.md:1]: ssd

Comment (bleow) [inline pages/Testing.md:1]: test

File: pages/Testing.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
+Just trying something new!
```
Comment (bleow) [inline pages/Testing.md:3]: mm

**Pull request closed without merging**